samples/doris-demo/spark-demo/org/apache/doris/scala/DorisSparkConnectionExampleScala.scala

```diff
@@ -0,0 +1,157 @@
+package org.apache.doris.scala
```

Comment:
add license head

Reply:
Usage documentation and license has been added.